spec/controllers/passwords_controller_spec.rb

```diff
@@ -111,7 +111,7 @@
           user_id: user,
         }
 
-        expect(response).to be_successful
+        expect(response).to have_http_status(:unprocessable_entity)
```

Comment:
@dwightwatson Can you add a similar test for the users_controller as well?

Reply:
Looks like the test case was missing, so I've added it now.